pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -142,6 +142,8 @@ type OperationExecutor interface {
 	IsOperationPending(volumeName v1.UniqueVolumeName, podName volumetypes.UniquePodName) bool
 	// Expand Volume will grow size available to PVC
 	ExpandVolume(*expandcache.PVCWithResizeRequest, expandcache.VolumeResizeMap) error
+	// VolumeFileSystemResize will resize volume's file system to expected size.
+	VolumeFileSystemResize(volumeToMount VolumeToMount, actualStateOfWorld ActualStateOfWorldMounterUpdater) error
```

Comment:
nit: Can we be consistent with naming here between `ExpandVolume` and `VolumeFileSystemResize`.  Also can the comments making it very clear what the difference between them is.

Reply:
Changed `VolumeFileSystemResize` to `ExpandVolumeFSWithoutUnmounting` and added a more clear comment.